openasip/src/applibs/LLVMBackend/LLVMTCECmdLineOptions.cc

```diff
@@ -433,13 +433,61 @@ LLVMTCECmdLineOptions::initialStackPointerValue() const {
 }
 
 bool
-LLVMTCECmdLineOptions::isDataStartAddressSet() const {
-   return findOption(SWL_DATA_START)->isDefined();
+LLVMTCECmdLineOptions::isDataStartAddressSet(
+    TTAMachine::AddressSpace& aSpace) const {
+    CmdLineOptionParser* opts = findOption(SWL_DATA_START);
+    if (opts->isDefined()) {
+        int size = opts->listSize();
+        if (size == 1) {
+            return true;
+        }
+        if (size % 1 != 0) {
```

Comment:
Is this always false?

Reply:
Yes it was always meant to check if the size is odd. Changed it to size % 2 != 0